### PR TITLE
Issue2624 space cooling tutorial

### PR DESCRIPTION
**Pull request by dhblum**
This is for https://github.com/lbl-srg/modelica-buildings/issues/2624 and implements the changes noted in that issue, along with documentation and reference result updates.

### CODE REVIEW

**Comment by dhblum** [comment]
@mwetter Still pending unit test passing, but this is otherwise ready for your review.

**Review by mwetter — APPROVED** [review]
This is ready to merge when all tests pass.

File: Buildings/Examples/Tutorial/SpaceCooling/System2.mo [hunk 1]
```diff
@@ -328,9 +328,9 @@ The calculations are as follows:
 
   /////////////////////////////////////////////////////////
   // Water temperatures and mass flow rates
-  parameter Modelica.SIunits.Temperature TWSup_nominal = 273.15+16
+  parameter Modelica.SIunits.Temperature TWSup_nominal = 289.15
```
Comment (dhblum) [inline Buildings/Examples/Tutorial/SpaceCooling/System2.mo:331]: @mwetter I'm realizing this supply/return specification is still backwards.  This is the documentation so shouldn't change the modeling results.

Comment (dhblum) [reply Buildings/Examples/Tutorial/SpaceCooling/System2.mo]: Fixed in https://github.com/lbl-srg/modelica-buildings/pull/2625/commits/f17f6363fc2b89f0a8332335300910b2f2e0761e.

**Comment by mwetter** [comment]
@dhblum : Can you please also update RoomHVAC.mo accordingly as it uses the old configuration and fails the CI test.

**Comment by dhblum** [comment]
@mwetter Ok I think I see what I need to do for that:
- Update ``Buildings.Fluid.FMI.ExportContainers.Examples.FMUs.HVACZone`` and parameterization of ``cooCoi``
- Update ``Buildings.Fluid.FMI.ExportContainers.Examples.FMUs.HVACZones`` and parameterization of ``cooCoi``
- Update unit test results
- Update plot figures in documentation

**Pull request merged**